common/decisions/hfe_decision_features.txt

```diff
@@ -102,6 +143,8 @@ hfe_decision_features_exodus = {
 					add_resource = {
 						energy = 50
 						minerals = 50
+						alloys = 25
+						influence = 1000
```

Comment:
too cheat

Reply:
Not true. The Exodus architects just loaded up the entire population onto a single fleet. Influence is political power. They have it in spades.

Reply:
You add 1k influence for every pop

Reply:
Okay, done